### ISSUE TITLE
Typo in clsim/python/traysegments/I3CLSimMakePhotons.py (Trac #2404)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2396]
The trunk of I3CLSimMakePhotons.py has a type of loggin instead of logging on line 314.  This is trivial to fix on versions you build yourself, but people using cvmfs meta-projects cannot run GEANT4 because of this error.  

Migrated from https://code.icecube.wisc.edu/ticket/2404
```json
{
    "status": "closed", 
    "changetime": "2020-01-29T18:43:34", 
    "description": "The trunk of I3CLSimMakePhotons.py has a type of loggin instead of logging on line 314.  This is trivial to fix on versions you build yourself, but people using cvmfs meta-projects cannot run GEANT4 because of this error.  ", 
    "reporter": "hignight", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1580323414413481", 
    "component": "combo core", 
    "summary": "Typo in clsim/python/traysegments/I3CLSimMakePhotons.py", 
    "priority": "trivial", 
    "keywords": "", 
    "time": "2020-01-27T22:29:46", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20200127T22:41:43: hignight commented:

> I should mention that this error is only in combo and not in the metaproject simulation.

**Comment by icecube-trac** [comment]
Trac update at 20200129T18:43:34: 
* jvansanten changed resolution from "" to "fixed"
* jvansanten commented:

> In changeset 178699:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="178699"
> Correct typo. Fixes #2396
> ```


* jvansanten changed status from "new" to "closed"
* jvansanten changed owner from "" to "jvansanten"
